README.md

```diff
@@ -62,16 +62,20 @@ feat(ui): Add `Button` component.
             fix
             feat
           # Configure which scopes are allowed (newline delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
           scopes: |
             core
             ui
+            (?![A-Z])+
```

Comment:
Should we maybe use the JIRA ID use case here? It seems like many users are interested in that.

Reply:
That's a good idea
Do you think something somewhat specific like `JIRA-\d+` would be sufficient? To draw the eye with the `JIRA` part (that people can replace for their own project)?

Reply:
Exactly, sounds perfect! 🙌

Reply:
Added the example